cluster-stamp.bicep

```diff
@@ -1214,47 +1214,6 @@ resource paDisallowNamespaceUsage 'Microsoft.Authorization/policyAssignments@202
   }
 }
 
-// Built-in policy: AKS clusters should gate deployment of vulnerable images' policy at the resource group level.
```

Comment:
what about keeping them in with a conditional? Not asking for a change just giving an alternative since it was originally intended to be included.

Reply:
We have some work to do to include the new Defender feature more completely.  Right now, these policies error out on evaluation when the feature isn't fully enabled -- including pricings in security center (which we don't configure here -- but do in the regulated).  Since we really aren't telling a defender story in the baseline, I'm only including the bits that don't cause Azure Policy failures to surface.  If we come back with bringing Defender into the baseline in a more "official" way, then this policy will need to come back.  This policy only really works if you've pushed all images through the new ACR pipeline audit feature -- which we very much don't do here.  Adding it as an optional param is probably a bit overkill at this point, since we don't set the user up for success with it.

Reply:
thanks for sharing the context and we can definitely get rid of it for now as this not adding value in the end.